pyhive/hive.py

```diff
@@ -49,6 +49,45 @@
 }
 
 
+def get_sasl_client(host, sasl_auth, service=None, username=None, password=None):
+    import sasl
+    sasl_client = sasl.Client()
+    sasl_client.setAttr('host', host)
+
+    if sasl_auth == 'GSSAPI':
+        sasl_client.setAttr('service', service)
+    elif sasl_auth == 'PLAIN':
+        sasl_client.setAttr('username', username)
+        sasl_client.setAttr('password', password)
+    else:
+        raise AssertionError
```

Comment:
s/AssertionError/ValueError and let's provide an error message

e.g. `ValueError("sasl_auth only supports GSSAPI and PLAIN")`

Reply:
done